pilot/cmd/pilot-agent/app/cmd.go

```diff
@@ -365,7 +365,7 @@ func applyExcludeInterfaces(ifaces []string) []string {
 				continue
 			}
 			unwrapAddr := ipAddr.Unmap()
-			if !unwrapAddr.IsValid() || unwrapAddr.IsLoopback() || unwrapAddr.IsLinkLocalUnicast() || unwrapAddr.IsLinkLocalMulticast() || unwrapAddr.IsUnspecified() {
```

Comment:
I am not sure why we donot exclude all these addresses

Reply:
Yeah, i mean since we excluded the nic, why donot we exclude all the ips associated with it.

Not relate to this patch though

Reply:
ohh yeah it's actually a good point to exclude all the IPs for excluded NIC. let me test that and I will update code.